test/conv.jl

```diff
@@ -639,3 +639,21 @@ end
         println()
     end
 end
+
+@testset "conv_wrapper" begin
+    x = rand(10, 10, 3, 10)
+    w = rand(2, 2, 3, 16)
+    w1 = rand(3, 4, 3, 16)
+    @test size(conv(x, w)) == (9, 9, 16, 10)
+    @test size(conv(x, w; stride = (2, 2), pad = (2, 2))) == (7, 7, 16, 10)
+    @test size(conv(x, w1; stride = (1, 2), pad = (2, 3))) == (12, 7, 16, 10)
+end
+
+@testset "depthwiseconv_wrapper" begin
+    x = rand(10, 10, 3, 10)
+    w = rand(2, 2, 3, 3)
+    w1 = rand(3, 4, 3, 3)
+    @test size(depthwiseconv(x, w)) == (9, 9, 9, 10)
+    @test size(depthwiseconv(x, w; stride = (2, 2), pad = (2, 2))) == (7, 7, 9, 10)
+    @test size(depthwiseconv(x, w1; stride = (1, 2), pad = (2, 3))) == (12, 7, 9, 10)
```

Comment:
One last nitpick; for both of these, let's exercise every parameter.  `stride`, `pad`, `dilation`, `flipped`.

Reply:
Sure. I have now added two more tests with all parameters.